src/repo_add_optionals.rs

```diff
@@ -0,0 +1,116 @@
+use std::process::Command;
+
```

Comment:
I wonder if this file should be called `repo_add_args` instead of `_optionals`. Seems like it'd be more consistent with the other filenames